ksqldb-execution/src/test/java/io/confluent/ksql/execution/util/ExpressionTypeManagerTest.java

```diff
@@ -1058,15 +1057,6 @@ public void shouldFailIfDefaultHasDifferentTypeToWhen() {
     ));
   }
 
-  @Test
-  public void shouldThrowOnTimeLiteral() {
-    // When:
```

Comment:
Are there any positive tests we can have in this class?

Reply:
Did you forget to push the changes? I don't see the test added

Reply:
whoops, pushed the changes 😬